clar2wasm/src/words/blockinfo.rs

```diff
@@ -316,6 +466,147 @@ mod tests {
                 .to_string()
                 .contains("expecting 2 arguments, got 3"));
         }
+
+        #[test]
+        fn get_tenure_info_less_than_two_args() {
+            let result = evaluate("(get-tenure-info? id-header-hash)");
+            assert!(result.is_err());
+            assert!(result
+                .unwrap_err()
+                .to_string()
+                .contains("expecting 2 arguments, got 1"));
+        }
+
+        #[test]
+        fn get_tenure_info_more_than_two_args() {
+            let result = evaluate("(get-tenure-info? id-header-hash u0 u0)");
+            assert!(result.is_err());
+            assert!(result
+                .unwrap_err()
+                .to_string()
+                .contains("expecting 2 arguments, got 3"));
+        }
+
+        // The hash here is taken from an older test for get-block-info?.
+        // Because the logic behind get-stacks-block-info? id-header-hash and
+        // get-block-info? id-header-hash are the same and both use the
+        // get_index_block_header_hash function the return value should be the
+        // same.
```

Comment:
Not sure that's the right position. It's an explanation for a part of the code, not the entire test.